code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -2493,3 +2493,165 @@
 /datum/sprite_accessory/ipc_chassis/zenghupharmaceuticals
 	name = "Zeng-Hu Pharmaceuticals"
 	limbs_id = "zhpipc"
+
+//apids
+
+/datum/sprite_accessory/apid_antenna
+	icon = 'icons/mob/apid_accessories.dmi'
+
+/datum/sprite_accessory/apid_antenna/moth
+	name = "mothlike antenna"
```

Comment:
Same with the capitalization here, other accessories are capitalized, I enjoy consistency

Reply:
addressed